man/picom.1.asciidoc

```diff
@@ -211,8 +211,8 @@ May also be one of the predefined kernels: `3x3box` (default), `5x5box`, `7x7box
 *--shadow-exclude-reg* 'GEOMETRY'::
 	Specify a X geometry that describes the region in which shadow should not be painted in, such as a dock window region.  Use `--shadow-exclude-reg x10+0-0`, for example, if the 10 pixels on the bottom of the screen should not have shadows painted on.
 
-*--xinerama-shadow-crop*::
-	Crop shadow of a window fully on a particular Xinerama screen to the screen.
+*--crop-shadow-to-monitor*::
+	Crop shadow of a window fully on a particular XRandR monitor to the monitor.
```

Comment:
I would potentially reword this sentence to not reference RandR as well:
`Crop shadow of a window fully on a particular monitor to that monitor.`
(should obviously be mirrored to the sample config and help message if to be changed.)

Reply:
personally, i'd preserve the mention of the xrandr here "to not leak the implementation details" as @yshui have said, so an end user could know that this option uses xrandr underneath and could poke around it in case of any issues.

Reply:
I would probably word it like: `Crop shadow of a window fully on a particular monitor to that monitor. This is currently implemented using the RandR extension`.

Reply:
@yshui, done.